haruka/modules/reporting.py

```diff
@@ -10,8 +10,10 @@
 from haruka.modules.log_channel import loggable
 from haruka.modules.sql import reporting_sql as sql
 from haruka.modules.tr_engine.strings import tld
+from haruka.modules.helper_funcs.chat_status import bot_admin, user_admin
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [redefinition of unused 'user_admin' from line 9 (F811)](https://app.codacy.com/gh/HarukaNetwork/HarukaAya/pullRequest?prid=5652154)

Reply:
oof... i forgot to remove the user_admin from line 9... if pushing to haruka aya master branch please remove it kthx